transformations/unit_converter/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "unit_converter",
+  "test_cases": [
+    {
+      "class": "UnitConverter",
+      "inputs": {
+        "sentence": "I weight 100 pounds."
+      },
+      "outputs": [{
+        "sentence": "I weight 1600.002 ounces."
+      }]
+    },
+    {
+      "class": "UnitConverter",
+      "inputs": {
+        "sentence": "I weight one-hundred pounds."
+      },
+      "outputs": [{
+        "sentence": "I weight one thousand, six hundred point zero zero two ounces."
```

Comment:
Each instance `I weight...` should be `I weigh...`

Reply:
Sorry, you are completely right, I have fixed them in 9c83eec73e6b1db51034a3e849e04fca9ac69032